charts/kuberpult/values.yaml

```
@@ -61,6 +61,16 @@ log:
   level: "WARN"
 cd:
   image: kuberpult-cd-service
+# In MOST cases, do NOT overwrite the "tag".
+# In general, kuberpult only guarantees that running with the same version of frontend and cd service will work.
+# For testing purposes, we allow to overwrite the tags individually, to test an old frontend service with a new cd service.
+  tag: "$VERSION"
+# By default, kuberpult allows names of apps to be max 39 characters.
+# This is usually enough. If you want to extend the length to up to 70 characters,
+# You need to ensure that ArgoCD has no issue with this. Set the Argo CD helm parameter:
+# `configs.cm.application.resourceTrackingMethod: annotation+label`
+# Note that there is no simple way in kuberpult to rename an app.
+  allowLongAppNames: true
```

The default should be false.
```suggestion
  allowLongAppNames: false
```

---

updated